plugin/png/src/test/java/it/geosolutions/imageioimpl/plugins/png/PNGWriterTest.java

```diff
@@ -16,52 +16,113 @@
  */
 package it.geosolutions.imageioimpl.plugins.png;
 
-import it.geosolutions.imageio.plugins.png.PNGWriter;
-import it.geosolutions.resources.TestData;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
 
 import java.awt.image.BufferedImage;
 import java.io.File;
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.OutputStream;
+import java.util.HashMap;
+import java.util.Map;
 
 import javax.imageio.ImageIO;
 
-import junit.framework.TestCase;
-
-import org.junit.Assert;
+import org.junit.Test;
 
 import ar.com.hjg.pngj.FilterType;
+import ar.com.hjg.pngj.PngReader;
+import ar.com.hjg.pngj.chunks.ChunksList;
+import ar.com.hjg.pngj.chunks.PngMetadata;
+import it.geosolutions.imageio.plugins.png.PNGWriter;
+import it.geosolutions.resources.TestData;
 
 /**
  * Unit test for simple App.
  */
-public class PNGWriterTest extends TestCase {
+public class PNGWriterTest {
 
-    public void testWriter(){
+    @Test
+    public void testWriter() {
         PNGWriter writer = new PNGWriter();
         OutputStream out = null;
-        try{
-            
-        // read test image
-        BufferedImage read = ImageIO.read(TestData.file(this, "sample.jpeg"));
-        
-        File pngOut = TestData.temp(this, "test.png",true);
-        out = new FileOutputStream(pngOut);
-        
-        writer.writePNG(read, out, 1, FilterType.FILTER_NONE);
-        BufferedImage test = ImageIO.read(pngOut);
-        Assert.assertNotNull(test);
-        }catch(Exception e){
+        try {
+
+            // read test image
```

Comment:
Are these tabs? :)

Reply:
Nope. just 4 spaces for better indentation. Previous code had all lines (included opening "try {" ) on the same column number.